andela/hc-beef-sauce
# [FT #158532838] Add departments to checks

### What does this PR do?
It allows for a user to add departments to a check as well as view the checks based on the department selected department.

### Description of Task to be completed?
Modify the 'update name' form and front views to accommodate adding departments to individual checks.
Update the checks table to hold departments values

### How should this be manually tested?
 - Select the update name text box
 - Input departments in the departments text box and click on save.
 - The added departments' buttons should then show up at the top of the checks  Clicking on one of these department buttons should then filter the displayed checks based on the selected department.

### Screenshots
<img width="1191" alt="screen shot 2018-07-18 at 14 10 01" src="https://user-images.githubusercontent.com/37480490/42878638-6c651da6-8a96-11e8-8890-ff0243e111e6.png">
Add new department to check

<img width="1258" alt="screen shot 2018-07-18 at 14 09 47" src="https://user-images.githubusercontent.com/37480490/42878652-79400194-8a96-11e8-8062-fe30d9add379.png">
